sqlglot/generator.py

```diff
@@ -12,6 +12,18 @@
 
 logger = logging.getLogger("sqlglot")
 
+UNESCAPED_SEQUENCE_TABLE = str.maketrans(
```

Comment:
this should be per dialect i think, everyone is different and configurable

Reply:
sounds good with me, shouldn't be much of a change